src/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -135,7 +132,7 @@ public static File getUserRootFolder() {
     final File userHome = new File(System.getProperties().getProperty("user.home"));
     // the default
     File rootDir;
-    if (GameRunner.isMac()) {
+    if (System.getProperties().getProperty("os.name").toLowerCase().contains("mac")) {
```

Comment:
@DanVanAtta Don't ask me why I can't use `GameRunner.isMac()` It keeps throwing Errors (not exceptions)